documentation/src/main/asciidoc/topics/con_discovery.adoc

```diff
@@ -0,0 +1,16 @@
+[id='cluster_discovery-{context}']
+= Cluster Discovery Protocols
+{brandname} supports different protocols that allow nodes to automatically find each other on the network and form clusters.
+
+{brandname} can use the following discovery mechanisms:
+
+* Generic discovery protocols (`TCPPING` and `TCPGOSSIP`)
+* JGroups PING protocols (`KUBE_PING` and `DNS_PING`)
```

Comment:
I don't understand this classification.
I'd say the discovery protocols fall into two broad categories: 
* the generic ones (`TCPPING` and `UDPPING`) which should work on most network and don't rely on external services
* those that rely on some external source for storing/retrieving topology information (DNS, Kubernetes, S3, JDBC, Gossip, etc)

Reply:
Thanks, that makes a lot more sense.

@tristantarrant I noticed there is no example of UDPPING in our docs. Do you mean PING instead? If so should we include an example of PING alongside the others?